src/redux/reducer.js

```diff
@@ -26,6 +45,19 @@ function reducer(originState = initialState, { type, payload }) {
                 ...originState,
                 filter: "SHOW_COMPLETED"
             }
+        case actionTypes.JUDGE_DOUBLE_CLICK:
+                    return {
+                        ...originState,
+                        showSideBar:true,
+                        showSideBarGetTodoid:payload
```

Comment:
`showSideBarGetTodoid` should be changed to a better name, this is the todo id which currently being edited.